3drepo/3drepo.io
# Issue 4832 - Milestone 2: 2d screenshot viewer

This fixes #4832 

#### Description
Opening a ticket in the viewer with an image (or a view) property offers the possibility take a screenshot of the drawing viewer if the latter is open.
The screenshot is taken using a service. This service has a `getScreenshot` function that is loaded by the drawing viewer upon changing drawing

#### Test cases
- Open the 3d viewer, but not the 2d one and try to take a 2d screenshot -> You shouldn't and a tooltip should tell you why
- Open the 2d viewer and try to take a screenshot -> You should see the markup option to edit the image
- Open the 2d viewer, move and zoom the image (try both svg and png) and try to take a screenshot -> You should the image ready for markup that matches what you see in the 2d viewer
